readme.md

```diff
@@ -91,18 +98,14 @@ Xin chào thế giới. This is Hello World in Vietnamese.
 \end{document}
 ```
 
-Here we use two packages `usepackage[T5]{fontenc}` and `usepackage[utf8]{inputenc}` . This is really simple to understand as the package will import font encoders to display your content correctly. If you are using TexMaker this is what the above code display :
-
-![](http://i.imgur.com/OUSPekM.png)
-
-vs without the packages :package::  
-
-![](http://i.imgur.com/Upd47xH.png)  
+The default font (Latin Modern) does not support all characters. You can,

```

Comment:
I am not professional in this so I am not gonna join the discussion here. I respect both of your opinions.  
One thing I'd like to mention is that here the repo focuses on simplicty and accessibility for majority of new-to-LaTeX people. Some problems may bother experts, some may not. Think carefully 👍